app/components/aboutMe/AboutMariana.js

```diff
@@ -0,0 +1,62 @@
+export default function About() {
+  return (
+    <>
```

Comment:
why is this an empty tag and not a div?

Reply:
okay :) 

but please use updated docs: https://react.dev/reference/react/Fragment#fragment